layers/meta-balena-fsl-arm/recipes-core/images/balena-image-initramfs.bbappend

```diff
@@ -3,4 +3,6 @@ PACKAGE_INSTALL:solidrun-n6g += " \
 "
 
 PACKAGE_INSTALL:remove:nitrogen8mm = " initramfs-module-recovery"
+PACKAGE_INSTALL:remove:imx8mm-lpddr4-evk = " initramfs-module-recovery"
```

Comment:
This was removed from the other device type because of space constraints. You should not be impacted by that since you enlarged the rootfs so you should be able to have this module included in the initramfs

Reply:
ok will add this back in!

Reply:
Have updated that now @floion